tests/st/utils/docker_host.py

```diff
@@ -216,7 +217,7 @@ def start_calico_node(self, options=""):
         Start calico in a container inside a host by calling through to the
         calicoctl node command.
         """
-        args = ['node', 'run']
+        args = ['node', 'run', '--node-image=%s' % NODE_CONTAINER_NAME]
```

Comment:
Also, there is a method `start_calico_node_with_docker` that should also be updated to use this container image.  We don't actually use it at the moment, but we should at least keep it updated so that we don't potentially run into problems if we do start using it.